app-backend/db/src/main/scala/com/azavea/rf/database/UserGroupRoleDao.scala

```diff
@@ -82,8 +82,13 @@ object UserGroupRoleDao extends Dao[UserGroupRole] {
   @SuppressWarnings(Array("OptionGet"))
   def createWithGuard(adminCheckFunc: (User, UUID) => ConnectionIO[Boolean], groupType: GroupType)
                      (groupId: UUID, actingUser: User, subjectId: String, userGroupRoleCreate: UserGroupRole.Create,
-                      platformId: UUID):
+                      platformId: UUID, isSameOrganizationIOO: Option[ConnectionIO[Boolean]]):
       ConnectionIO[UserGroupRole] =
+  {
```

Comment:
I think the new argument for this function should not be an `Option` -- if an `IO` for checking if a user is in the same organization is set to `None` in cases where the `Option` is empty, then setting the default to be `false.pure[ConnectionIO]` seems reasonable in my opinion to avoid this extra block of code that is kind of confusing at first glance (initially I was left figuring out what the difference is between `isSameOrgIO` and the argument `isSameOrganizationIO` since the names are so similar).

What do you think?

Reply:
I think it is a good point. I'll update it.